modules/ROOT/pages/authentication-authorization/dbms-administration.adoc

```diff
@@ -697,7 +787,7 @@ To list all privileges for the role `statusModifier` as commands, use the follow
 
 [source, cypher, role=noplay]
 ----
-SHOW ROLE statusModifier PRIVILEGES AS COMMANDS
+SHOW ROLE statusModifier PRIVILEGES AS COMMANDS;
```

Comment:
we shouldn't use `statusModifier` in this example, not if we want it to only deal with home db since that is the role that was used for status, should be updated to something like `homeDbModifier` (and added to test setup at top of page)